nodes/If.py

```diff
@@ -1,2 +1,24 @@
-if 3 > 2:
+"""
+If astroid node
+
+An if statement.
+
+Attributes:
+    - test    (Node)
+        - Holds a single node such as Compare.
+    - Body    (List[Node])
+        - A list of nodes that will execute if the condition passes.
+    - orelse  (List[Node])
+        - The else clause.
```

Comment:
This one is worth expanding on to incorporate an example with `elif`. Such clauses are represented in an interesting way in astroid.